exp/views/study.py

```diff
@@ -2782,7 +2888,12 @@ def get_permitted_triggers(view_instance, triggers):
             # 3) trigger is found in user's study permissions
         if not user.is_superuser:
             if trigger in admin_triggers:
-                if not (user.organization == study.organization and user.is_org_admin):
+                if not (
+                    user.lab == study.lab
+                    and user.has_perm(
+                        LabPermission.CHANGE_STUDY_STATUS.codename, user.lab
+                    )
+                ):
```

Comment:
I'm a little confused about what this section is doing and why the admin_triggers (approve, reject) are being broken out separately for non-superusers. This seems to say that you can't cause a study to be approved/rejected unless you have the *lab* level permission specifically, rather than either lab or study level (as for e.g. can_edit_study). Also why include 'approve' which is never granted to non-superusers?

Reply:
I think for this particular section I had just changed some of the organization stuff and replaced it with lab stuff, just to get the view serving.

If something looks really out of place, it's probably that sort of thing.